src/services/api.ts

```diff
@@ -2,7 +2,7 @@ import axios from 'axios';
 import { devPrint } from '../components/utils/RandomUtils';
 
 const api = axios.create({
-  baseURL: 'https://api.swecc.org',
+  baseURL: 'http://localhost:8000/', //'https://api.swecc.org',
```

Comment:
Do you want to just make this a check on prod/dev so we don't change the url for local dev work? This needs to be changed back anyways for this to work.

Reply:
yeah I add